administration/src/bp-modules/applications/__tests__/VerificationsView.test.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { act } from 'react'
+
+import { renderWithTranslation } from '../../../testing/render'
+import VerificationsView from '../VerificationsView'
+import { verificationsMixed } from '../__mocks__/verificationData'
+
+describe('VerificationsView', () => {
+  it('should show a hint if there are no verifications', async () => {
+    const { getByTestId, getByText } = renderWithTranslation(<VerificationsView verifications={[]} />)
+    await act(async () => null) // Blueprint5.Icon update
+    expect(getByText('Bestätigung(en) durch Organisationen:')).toBeTruthy()
+    expect(getByTestId('no-verifications-hint').textContent).toBe('(keine)')
+  })
+
+  it('should render a list of verification items', async () => {
+    const { getAllByTestId, getByText } = renderWithTranslation(
+      <VerificationsView verifications={verificationsMixed} />
+    )
+    await act(async () => null) // Blueprint5.Icon update
+    expect(getByText('Bestätigung(en) durch Organisationen:')).toBeTruthy()
+    expect(getAllByTestId('verification-list-item')).toHaveLength(3)
```

Comment:
@steffenkleinle i know that it also should be possible to use here 
```
 expect(queryAllByRole('li')).toHaveLength(3)
 ```
 But for me often it does not work, even the items are in the dom